2009-09-22  Stepan Kasal  <skasal@redhat.com>

	* tell aclocal where are the macros

--- libtar-1.2.11/configure.ac.macro	2009-09-22 14:47:07.000000000 +0200
+++ libtar-1.2.11/configure.ac	2009-09-22 14:50:34.000000000 +0200
@@ -2,6 +2,7 @@
 AC_INIT([libtar], [1.2.11])
 AC_PREREQ([2.57])
 AC_CONFIG_AUX_DIR([autoconf])
+AC_CONFIG_MACRO_DIR([autoconf])
 AC_CONFIG_HEADERS([config.h])
 AC_COPYRIGHT([[
 Copyright (c) 1998-2003 University of Illinois Board of Trustees
